src/astro/sql/operators/transform.py

```diff
@@ -96,3 +98,41 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
         decorated_operator_class=TransformOperator,
         **kwargs,
     )
+
+
+def transform_file(
+    file_path: str,
+    conn_id: str = "",
+    parameters: Mapping | Iterable | None = None,
+    database: str | None = None,
+    schema: str | None = None,
+    **kwargs: Any,
+) -> XComArg:
+    """
```

Comment:
@sunank200 Can you add a 1-line description here in your PR too about what this function is doing

Reply:
sure